docs/cni-plugin.md

```diff
@@ -45,6 +45,35 @@ Another example with a trunk port and jumbo frames:
 * `mtu` (integer, optional): MTU.
 * `trunk` (optional): List of VLAN ID's and/or ranges of accepted VLAN
   ID's.
+* `configuration_path` (optional): configuration file containing ovsdb
+  socket file path, etc.
+
+### Flatfile Configuation
+
+There is one option for flat file configuration:
+
+* `configuration_path`: A file path to a OVS CNI configuration file.
+
+OVS CNI will look for the configuration in these locations, in this order:
+
+* The location specified by the `configuration_path` option.
+* `/etc/kubernetes/cni/net.d/ovs.d/ovs.conf`
+* `/etc/cni/net.d/ovs.d/ovs.conf`
+
+You may specify the `configuration_path` to point to another location should it be desired.
+
+Any options added to the `ovs.conf` are overridden by configuration options that are in the
+CNI configuration (e.g. in a custom resource `NetworkAttachmentDefinition` used by Multus CNI
+or in the first file ASCII-betically in the CNI configuration directory -- which is
```

Comment:
"ASCII-betically"

:D

Reply:
done

Reply:
Oh, that was not a request for change, I just liked the way you name it :)

Reply:
ok ;) actually this doc and implementation approach inspired from [whereabouts cni](https://github.com/openshift/whereabouts-cni#flatfile-configuration)!

Reply:
Whereabouts rocks!